### PR TITLE
Added internal method to validate if connecting server is UCS

### PR DESCRIPTION
**Pull request by ragupta-git**
Added internal method to validate if connecting server is UCS.

### CODE REVIEW

File: ucsmsdk/ucssession.py [hunk 1]
```diff
@@ -450,14 +472,8 @@ def _login(self, auto_refresh=False, force=False):
...
-        nw_elem = config_resolve_class(cookie=self.__cookie,
-                                          in_filter=None,
-                                          class_id="networkElement")
-        nw_elem_response = self.post_elem(nw_elem)
-        # To Do - returns error tag
-        if nw_elem_response.error_code != 0:
-            self.__clear()
-            return False
+        if not self.__validate_ucsm():
+            raise UcsLoginError("Invalid UCS")
```
Comment (vvb) [inline ucsmsdk/ucssession.py]: "Invalid UCS" should be "Not a supported server"

File: ucsmsdk/ucssession.py [hunk 2]
```diff
@@ -392,6 +392,29 @@ def _refresh(self, auto_relogin=False):
...
+
+        from .ucsmethodfactory import config_resolve_class
+
+        nw_elem = config_resolve_class(cookie=self.__cookie,
+                                       in_filter=None,
+                                       class_id="networkElement")
+        try:
+            nw_elem_response = self.post_elem(nw_elem)
+            if nw_elem_response.error_code != 0:
+                self._logout()
```
Comment (vvb) [inline ucsmsdk/ucssession.py]: should this be `self._logout()` or `self._clear()`. The older code was doing a clear. Just want to double check

Comment (ragupta-git) [inline ucsmsdk/ucssession.py]: Yes it should be logout as we already did the login before checking for the server.

File: ucsmsdk/ucssession.py (hunk 1 above)
Comment (ragupta-git) [inline ucsmsdk/ucssession.py]: Done.

**Pull request merged**